.env

```diff
@@ -0,0 +1,7 @@
+REACT_APP_FIREBASE_API_KEY=AIzaSyAsJ3-W1TEP6sFnqFwO1cpVRj4BQgoEc-0
```

Comment:
remove this! should not be uploading .env files to a public repo
Dont forget to regenerate the keys

Reply:
shoot, will do. i thought i'd added it to gitignore.